packages/@aws-cdk/aws-servicecatalogappregistry/lib/target-application.ts

```diff
@@ -108,6 +120,7 @@ class CreateTargetApplication extends TargetApplication {
 
     return {
       application: appRegApplication,
+      enableCrossAccountStacks: this.applicationOptions.enableCrossAccountStacks ?? false,
```

Comment:
Why do we have so many places where this default needs to be set?

Reply:
Is there a way we can configure their accounts to be able to do cross-account sharing if it isn't set up correctly?

Reply:
> Why do we have so many places where this default needs to be set?

We need to set this in both TargetApplication sub-classes CreateTargetApplication and ExistingTargetApplication to set a default if the ApplicationAssociator user has not specified it, and then a separate flag has to be handled in StackAssociatorBase for defining the default behavior for scenarios that don't involve ApplicationAssociator.

> Is there a way we can configure their accounts to be able to do cross-account sharing if it isn't set up correctly?

There could be a way to configure within CDK to automatically manage the AWS Organization membership of an account, but for now we are limiting ApplicationAssociator to focus more on the AppRegistry application associations. For now, outside of CDK, each account will need to take action to be part of the same AWS Organization as a "Management" account, and then the Management account will need to enable cross-account sharing.